crank/models.py

```diff
@@ -1,11 +1,62 @@
 from django.db import models
+from django.contrib.auth.models import User
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
 
 # Create your models here.
 class Greeting(models.Model):
     when = models.DateTimeField('date created', auto_now_add=True)
 
+class Profile(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    email_confirmed = models.BooleanField(default=False)
+
+#begin Tony-Frank pair insert
+class Course_Listing_Table(models.Model):
+    id = models.IntegerField(primary_key=True)
```

Comment:
Django will provide this field by default if you Take this line out and it will be auto incremented as well.
Same for the other id fields. 

Unless there’s a reason we want to supply our own ids when we save these records